docs/INTERFACE.md

```diff
@@ -0,0 +1,76 @@
+# Virtual machine Library
+The virtual machine library provides the following interfaces:
+
+` uintptr_t linux_setup_images(uintptr_t` *ram\_start*,
```

Comment:
I don't think any of the `\_` are necessary.

Instead for a big function like this we can wrap it in '```'

Reply:
Some markdown interpreters (including the one I was using to edit) don't distinguish between `_` and `*` and end the italic prematurely